plugins/woocommerce/src/Blocks/BlockTypes/EmailContent.php

```
@@ -0,0 +1,54 @@
+		return null;
+	}
+
+	/**
+	 * Renders Woo content placeholder to be replaced by content during sending.
+	 *
+	 * @param array     $attributes Block attributes.
+	 * @param string    $content Block content.
+	 * @param \WP_Block $block Block instance.
+	 * @return string Rendered block output.
+	 */
+	protected function render( $attributes, $content, $block ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attributes'. (Unused Code Rules)

(UnusedFormalParameter)